sky/packages/sky/lib/src/material/bottom_sheet.dart

```diff
@@ -22,7 +24,7 @@ class _BottomSheet extends StatefulComponent {
   }) : super(key: key);
 
   final Widget child;
-  final PerformanceView performance;
+  final Performance performance;
```

Comment:
Not sure what you mean. I think I'm doing roughly what Drawer does vis the performance.